A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/CDU/A320_Neo_CDU_AirwaysFromWaypointPage.js

```diff
@@ -21,17 +23,24 @@ class A320_Neo_CDU_AirwaysFromWaypointPage {
                 CDUFlightPlanPage.ShowPage(mcdu, 0);
             });
         };
+        allRows.forEach((r, idx) => {
+            if (r[0] != "" && r[1] != "") {
+                subRows[idx] = ["VIA", "TO"];
+            }
+        });
+        mcdu._titleElement.innerHTML = `<span><span>AIRWAYS</span> <span class='s-text'>FROM </span><span class='green'>${waypoint.ident}</span></span>`;
         let showInput = false;
         const departureWaypoints = mcdu.flightPlanManager.getDepartureWaypoints();
         const routeWaypoints = mcdu.flightPlanManager.getEnRouteWaypoints();
-        offset = routeWaypoints.indexOf(waypoint) >= 0 ? routeWaypoints.indexOf(waypoint) + (departureWaypoints.length ? 1 : 0) + 1 : offset;
+        // offset = routeWaypoints.indexOf(waypoint) >= 0 ? routeWaypoints.indexOf(waypoint) + (departureWaypoints.length ? 1 : 0) + 1 : offset;
```

Comment:
Can this not be done using the "[color>blue" pattern I see around h ere ?

Reply:
I tried to do it with that approach but I was not able to have multiples color configuration on the same string for the title.
I'm probably missing something on the approach, if you have any reference please let me know.

Reply:
We'll probably add that possibility at a later time, good for now